src/src/tx_main.cpp

```diff
@@ -474,14 +475,34 @@ void ICACHE_RAM_ATTR SendRCdataToRF()
   Radio.TXdataBuffer[0] = (Radio.TXdataBuffer[0] & 0b11) | ((crc >> 6) & 0b11111100);
   Radio.TXdataBuffer[7] = crc & 0xFF;
 
+#if defined(Regulatory_Domain_EU_CE_2400)
+  // Emulate that TX just happened, even if it didn't because channel is not clear
+  if (ChannelIsClear())
+  {
```

Comment:
Because LBTChannelBusy is gone, you can just wrap the `if` line with the defined now so 2x Radio.TXnb aren't needed.
```
#if defined(Regulatory_Domain_EU_CE_2400)
  // Emulate that TX just happened, even if it didn't because channel is not clear
  if (ChannelIsClear())
#endif
  {
    Radio.TXnb();
  }
```

Reply:
Thanks! Implemented!